src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -380,6 +380,9 @@
   <data name="CloseWindowCommandKey" xml:space="preserve">
     <value>Close window</value>
   </data>
+  <data name="ToggleFocusModeCommandKey" xml:space="preserve">
+    <value>Toggle focus mode</value>

```

Comment:
please add a `<comment>` explaining what "focus" means in this context. It might help our translators choose the appropriate sentiment :smile: